Sources/xdm/XDMFormatters.swift

```diff
@@ -22,6 +22,7 @@ public enum XDMFormatters {
     ///   - Date: A timestamp and it must not be null
     /// - Returns: The timestamp formatted to a string in the format of 'yyyy-MM-dd'T'HH:mm:ssXXX',
     ///            or an empty string if Date  is null
+    @available(*, deprecated, message: "Use function getISO8601UTCDateWithMilliseconds() in Date class extension from AEPServices module instead.")
```

Comment:
This seems ok to me. I wish we could have used the renamed attribute here but the usage is slightly different and in a different module.

Reply:
I played around with `renamed` but it appears the method needs to be in the same class as that of the deprecated. Plus, as you said, the replacement is an extension to Date and not a utility function, so the usage is different.
There's also the `obsoleted` tag, but it appears that applies to either a Swift or iOS version, and not the SDK version, so that doesn't make much sense to use in this case.